packages/bot-web-ui/src/components/dashboard/dashboard-component/load-bot-preview/bot-preview.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import WorkspaceControl from './workspace-control';
+
+type TBotPreview = {
+    id_ref: HTMLElement | React.ReactNode | null;
+};
+
+const BotPreview = ({ id_ref }: TBotPreview) => {
+    return (
+        <div
+            className='load-strategy__preview-workspace-container'
+            id={'load-strategy__blockly-container'}
```

Comment:
remove the extra braces since we can pass id in single quotes
```suggestion
            id='load-strategy__blockly-container'
```

Reply:
done ✅